File: ExxercicioFixacao2-Restaurante/ExxercicioFixacao2/ViewController.m
//
//  ViewController.m
//  ExxercicioFixacao2
//
//  Created by Usuário Convidado on 01/04/19.
//  Copyright © 2019 paulinho. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (IBAction)btnPagar:(id)sender {
    Cliente *c=[[Cliente alloc]init];
    float resultado = [c pagarContaComValorConta:self.txtConta.text.floatValue eValorDinheiro:self.txtPago.text.floatValue];
    self.txtTroco.text = [NSString stringWithFormat:@"%0.2f", resultado];
     
    
}

- (IBAction)btnReservar:(id)sender {
     Cliente *c=[[Cliente alloc]init];
    [c ReservarMesaComHorario:20];
}
@end

File: PauloSthivenGomesDaSilva_RM81631_MODELO_1/PauloSthivenGomesDaSilva_RM81631_MODELO_1/ViewController.swift
//
//  ViewController.swift
//  PauloSthivenGomesDaSilva_RM81631_MODELO_1
//
//  Created by Usuário Convidado on 13/05/19.
//  Copyright © 2019 Paulo Sthiven Gomes Da Silva. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBOutlet weak var txtIdade: UITextField!
    
    @IBOutlet weak var txtDiasVividos: UITextField!
    
    @IBAction func btnCalcular(_ sender: Any) {
        let idade = Int(txtIdade.text!);
        
        let resultado = idade! * 365;
        
        txtDiasVividos.text = String(resultado);
    }
}

